anchore_engine/services/policy_engine/engine/tasks.py

```diff
@@ -221,15 +224,14 @@ def execute(self):
         start_time = datetime.datetime.utcnow()
         try:
             start_time = datetime.datetime.utcnow()
-            updated_dict = DataFeeds.sync(
+            updated_list = DataFeeds.sync(
```

Comment:
Nit: Maybe change this to something like updated_data_feeds the name won't ever have to change again at some point to reflect a different data structure.

Reply:
@dspalmer99 that is a good point, in Python it is usually not a good pattern to use the type as part of the variable. As you pointed out, a changing type is one reason

Reply:
suggestion incorporated